beacon-chain/sync/subscriber_beacon_blocks_test.go

```diff
@@ -194,3 +194,12 @@ func TestReconstructAndBroadcastBlobs(t *testing.T) {
 		})
 	}
 }
+
+func TestReconstructAndBroadcastBlobs_Capella(t *testing.T) {
```

Comment:
This test doesn't really check anything. It will pass even if you reverted your change. You can theoretically check that some error logs are not present, but I am not sure if it's worth it.

Reply:
We don't want to log errors or debug. This would create verbose logs for nodes in Capella. The most I'd consider logging is at the trace level, but honestly, I don't think it's worth logging. I'm leaning towards keeping the original state as it is